tests/generative/variable_strategies.py

```diff
@@ -616,14 +623,18 @@ def valid_population(draw):
         hyp.assume(is_valid_population(population))
         return population
 
-    return st.builds(make_dataset, valid_population(), valid_variable())
+    return st.builds(
+        make_dataset, valid_population(), valid_patient_variable(), valid_event_series()
+    )
 
 
-def make_dataset(population, variable):
+def make_dataset(population, patient_variable, event_series):
     return Dataset(
         population=population,
-        variables={"v": variable},
-        events={},
+        variables={"v": patient_variable},
+        events={
+            "event_table": CombinedSeriesFrame({"e": event_series}),
+        },
```

Comment:
This means that we only run hypothesis tests with an event table. Do we also need to allow for there to be no event tables sometimes? Or is it sufficient that this is covered in tests elsewhere?

Reply:
That's a good point. I think we can get Hypothesis to decide whether or not to include one without too much extra effort.